crates/astria-cli/src/sequencer/ics20_withdrawal.rs

```diff
@@ -0,0 +1,119 @@
+use astria_core::{
+    primitive::v1::{
+        asset,
+        Address,
+    },
+    protocol::transaction::v1alpha1::{
+        action::Ics20Withdrawal,
+        Action,
+    },
+};
+use color_eyre::eyre::{
+    self,
+    WrapErr as _,
+};
+use ibc_types::core::{
+    channel::ChannelId,
+    client::Height,
+};
+use tracing::info;
+
+use crate::utils::submit_transaction;
+
+fn now_plus_5_minutes() -> u64 {
+    use std::time::Duration;
+    tendermint::Time::now()
+        .checked_add(Duration::from_secs(300))
+        .expect("adding 5 minutes to the current time should never fail")
+        .unix_timestamp_nanos()
+        .try_into()
+        .expect("timestamp must be positive, so this conversion would only fail if negative")
+}
+
+#[derive(clap::Args, Debug)]
+pub(super) struct Command {
+    /// The transfer amount to send
+    #[arg(long)]
+    amount: u128,
+    /// The address on the destination chain
+    #[arg(long)]
+    destination_chain_address: String,
+    /// The source channel used for withdrawal
+    #[arg(long)]
+    source_channel: String,
+    /// The address to refund on timeout, if unset refunds the signer
+    #[arg(long)]
+    return_address: Address,
```

Comment:
We'd need to make this field an `Option` to allow it to be unset.  Just now, the command will fail if `--return-address` is not specified.

Reply:
Should we set the return address to the sender address by default if `--return-address` is not specified?

Reply:
Makes sense to me, yes.  But if others disagree, we need to update the comment to not say ", if unset refunds the signer".